source/threads.tex

```diff
@@ -2406,417 +2406,418 @@
-  constexpr bool atomic_flag_test(const atomic_flag*) noexcept;                     // freestanding
-  bool atomic_flag_test_explicit(const volatile atomic_flag*,                       // freestanding
+  bool atomic_flag_test(const volatile atomic_flag*) noexcept;
+  constexpr bool atomic_flag_test(const atomic_flag*) noexcept;
+  bool atomic_flag_test_explicit(const volatile atomic_flag*,
                                  memory_order) noexcept;
-  constexpr bool atomic_flag_test_explicit(const atomic_flag*,                      // freestanding
+  constexpr bool atomic_flag_test_explicit(const atomic_flag*,
                                            memory_order) noexcept;
-  bool atomic_flag_test_and_set(volatile atomic_flag*) noexcept;                    // freestanding
-  constexpr bool atomic_flag_test_and_set(atomic_flag*) noexcept;                   // freestanding
-  bool atomic_flag_test_and_set_explicit(volatile atomic_flag*,                     // freestanding
+  bool atomic_flag_test_and_set(volatile atomic_flag*) noexcept;
+  constexpr bool atomic_flag_test_and_set(atomic_flag*) noexcept;
+  bool atomic_flag_test_and_set_explicit(volatile atomic_flag*,
                                          memory_order) noexcept;
-  constexpr bool atomic_flag_test_and_set_explicit(atomic_flag*,                    // freestanding
+  constexpr bool atomic_flag_test_and_set_explicit(atomic_flag*,
                                                    memory_order) noexcept;
-  void atomic_flag_clear(volatile atomic_flag*) noexcept;                           // freestanding
-  constexpr void atomic_flag_clear(atomic_flag*) noexcept;                          // freestanding
-  void atomic_flag_clear_explicit(volatile atomic_flag*, memory_order) noexcept;    // freestanding
-  constexpr void atomic_flag_clear_explicit(atomic_flag*, memory_order) noexcept;   // freestanding
-
-  void atomic_flag_wait(const volatile atomic_flag*, bool) noexcept;                // freestanding
-  constexpr void atomic_flag_wait(const atomic_flag*, bool) noexcept;               // freestanding
-  void atomic_flag_wait_explicit(const volatile atomic_flag*,                       // freestanding
+  void atomic_flag_clear(volatile atomic_flag*) noexcept;
+  constexpr void atomic_flag_clear(atomic_flag*) noexcept;
+  void atomic_flag_clear_explicit(volatile atomic_flag*, memory_order) noexcept;
+  constexpr void atomic_flag_clear_explicit(atomic_flag*, memory_order) noexcept;
+
+  void atomic_flag_wait(const volatile atomic_flag*, bool) noexcept;
+  constexpr void atomic_flag_wait(const atomic_flag*, bool) noexcept;
+  void atomic_flag_wait_explicit(const volatile atomic_flag*,
                                  bool, memory_order) noexcept;
-  constexpr void atomic_flag_wait_explicit(const atomic_flag*,                      // freestanding
+  constexpr void atomic_flag_wait_explicit(const atomic_flag*,
```

Comment:
Now that we no longer have those comments, could you please do another pass over all the line breaks and see if they can go now?

Reply:
Done